app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -94,14 +95,20 @@ app.views.Hovercard = app.views.Base.extend({
 
   _populateHovercard: function() {
     var href = this.href();
-    href += "/hovercard.json";
+    href += ".json";
```

Comment:
I agree with @cmrd-senya, having a `Hovercard` method inside the `PersonPresenter` feels nicer for me.